main.go

```diff
@@ -94,6 +103,62 @@ func showErrorMessages(action string, errorMessages map[string]error) error {
 	return nil
 }
 
+type ConfProfile struct {
+	RemoteConfiguration bool `json:"active"`
+	Compliance          bool `json:"compliance"`
+	Insights            bool `json:"insights"`
+	Remediation         bool `json:"remediations"`
+}
+
+// getConfProfile will retrieve the profile the system is configured
+func getConfProfile(client *http.Client) (*ConfProfile, error) {
+	var profile ConfProfile
+
+	url, err := GuessAPIURL()
+	if err != nil {
+		return nil, err
+	}
+
+	res, err := client.Get(url)
+	if err != nil {
+		return nil, err
+	}
+	// Read the response body
```

Comment:
Technically, this doesn't read the response body. This ensures that the response body (an `io.Reader`) is closed after opening by calling its `Close` method in the defer block of the function.

Reply:
You are right, I put the comment before adding the lines. I have move it to the place the body is read.